src/shared/lib/oql/accessors.ts

```diff
@@ -125,7 +125,7 @@ export default class accessors {
 
     public mut_amino_acid_change(d: Mutation) {
         if (this.alterationType(d.molecularProfileId) === 'MUTATION_EXTENDED') {
```

Comment:
we need to refactor oqlFilter.js to typescript and deal with the null case in some fashion.